src/PreCharge/PreCharge.cpp

```diff
@@ -78,10 +81,18 @@ void PreCharge::handle() {
 }
 
 void PreCharge::getSTO() {
+    uint8_t* gfdBuffer;
```

Comment:
This should instead instantiate a `uint8_t` and pass `&gfdBuffer` to `requestIsolationState()` because if you just declare a `uint8_t*`, you don't know where it's pointing and could overwrite something.